packages/admin/cms-admin/src/generator/utils/find-root-blocks.ts

```diff
@@ -0,0 +1,61 @@
+import { existsSync } from "fs";
+import { IntrospectionObjectType, IntrospectionQuery } from "graphql";
+
+import { CrudGeneratorConfig } from "../types";
+
+const fallbackLibraryBlocks: { [key: string]: string } = {
+    DamImageBlock: "@comet/cms-admin",
+};
+
+export function findRootBlocks({ entityName, ...generatorOptions }: CrudGeneratorConfig, schema: IntrospectionQuery) {
+    const ret = generatorOptions.rootBlocks || {};
```

Comment:
Do we still need this option? You're not using it in Demo right now

Reply:
The heuristics in findRootBlocks might not work correclty, that's why I left that. Do you think I should remove it?